### ISSUE TITLE
`pac solution sync` fails for --packagetype Both

### ISSUE THREAD

**Issue by filcole** [microsoft/powerplatform-vscode#261]
When trying out `pac solution sync` with `--packagetype Both` I get a crash immediately after the managed solution is exported with error:

```
   Error: Assumed Managed solution file not found: C:\Users\XXXX\AppData\Local\Temp\ylplyyig.1gd\DevExample_managed.zip
```

In the log below I clone, cd into the exported folder and then immediately sync.

I'm using pac 1.17.4.   The above works fine with `--packagetype Unmanaged`

```text
PS C:\Dev\pacbuild> dir

    Directory: C:\Dev\pacbuild

Mode                 LastWriteTime         Length Name    
----                 -------------         ------ ----    
d----          12/08/2022    23:17                myplugin

PS C:\Dev\pacbuild> pac solution clone --packagetype Both --name DevExample
Connected to...Phil Cole's Environment
Starting Solution Export...
Unmanaged solution export succeeded.
Starting Solution Export...
Managed solution export succeeded.
Unpacking Solution...

Extracting C:\Users\XXXX\AppData\Local\Temp\DevExample.zip and C:\Users\XXXX\AppData\Local\Temp\DevExample_managed.zip to C:\Dev\pacbuild\DevExample\src

Skipping localization
Processing Component: Entities    
 - Account
Processing Component: WebResources
 - pgc_/img/AccountNumber.png     
Processing Component: SolutionPluginAssemblies
 - myplugin, Version=1.0.0.0, Culture=neutral, PublicKeyToken=7fd009ec86136436
Processing Component: SdkMessageProcessingSteps
 - 58f66267-901a-ed11-b83e-000d3adaf6ec
Processing Component: AppModuleSiteMaps
 - pgc_AccountNumberGeneration
Processing Component: AppModules
 - pgc_AccountNumberGeneration

Unmanaged Extract complete.

Skipping localization
Processing Component: Entities
 - Account
Processing Component: AppModuleSiteMaps
 - pgc_AccountNumberGeneration
Processing Component: AppModules
 - pgc_AccountNumberGeneration

Managed Extract complete.

Unpacked Solution.
Solution clone extract succeeded.
Dataverse solution project with name 'DevExample' created successfully in: 'C:\Dev\pacbuild'
PS C:\Dev\pacbuild> cd .\DevExample\
PS C:\Dev\pacbuild\DevExample> dir

    Directory: C:\Dev\pacbuild\DevExample

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          13/08/2022    00:50                src
-a---          12/08/2022    22:42            129 .gitignore
-a---          13/08/2022    00:50           2474 DevExample.cdsproj

PS C:\Dev\pacbuild\DevExample> pac solution sync --packagetype Both
Connected to...Phil Cole's Environment
Starting Solution Export...
Unmanaged solution export succeeded.
Starting Solution Export...
Managed solution export succeeded.
Unpacking Solution...
Microsoft PowerPlatform CLI
Version: 1.17.4+g2bdb4f2

Error: Assumed Managed solution file not found: C:\Users\XXXX\AppData\Local\Temp\ylplyyig.1gd\DevExample_managed.zip


Usage: pac solution sync [--solution-folder] [--include] [--async] [--max-async-wait-time] [--processCanvasApps] [--packagetype]

  --solution-folder           Path to the local, unpacked solution folder: either the root of the 'Other/Solution.xml' file or a folder with a .cdsproj file. (alias: -f)    
  --include                   Which settings should be included in the solution being exported (alias: -i)
                              Values: autonumbering, calendar, customization, emailtracking, externalapplications, general, isvconfig, marketing, outlooksynchronization, relationshiproles, sales
  --async                     Exports solution asynchronously (alias: -a)
  --max-async-wait-time       Max asynchronous wait time in minutes. Default value is 60 minutes (alias: -wt)
  --processCanvasApps         (Preview) Pack/unpack any Canvas apps (.msapp) while processing the solution. default: false (alias: -pca)
  --packagetype               When unpacking/extracting, use to specify dual Managed and Unmanaged operation. When packing, use to specify Managed or Unmanaged from a previous unpack 'Both'. Can be: 'Unmanaged', 'Managed' or 'Both'; default: 'Unmanaged' (alias: -p)
```

**Comment by davidjenni** [comment]
tracking as AB#2873281

**Comment by davidjenni** [comment]
shipped as part of the August refresh, 1.18.4
